### ISSUE TITLE
bw.Shear.TimeOfDay - Allow for seasonal variation in dawn/dusk times

### ISSUE THREAD

**Issue by AndyBrightWind** [brightwind-dev/brightwind#336]
Function is most often used to derive shear by day and night separately. Currently, this can be done with 2 segments and a single time applied throughout the year. In reality, times of dawn and dusk vary through the year. It would be great to be able to input a list of daily dawn and dusk times throughout the year to allow this be captured more fully. Would also need the same functionality when applying shear object by day/night.

**Comment by BiancaMorandi** [comment]
The two fixes/commits done by Michael can be ignored as they have been addressed in #347

**Comment by stephenholleran** [comment]
@BiancaMorandi great.
Do you want me to close this issue and move it to Done?

**Comment by BiancaMorandi** [comment]
@stephenholleran no, the changes I made in #347 are not specific to this issue (but are changes that Michael had already addressed as part of this branch). I am going to delete the branch and then we will create a new one when addressing the actual issue.
